app/components-react/sidebar/SideNav.m.less

```diff
@@ -1,147 +1,217 @@
+@import '../../styles/index';
 @import '../../styles/mixins';
 @import '../../styles/badges';
 
-.container {
-  position: relative;
-  display: flex;
-  flex-direction: column;
-  height: 100%;
-  width: 52px;
-  left: 0;
-  top: 0;
-  bottom: 0;
-  background: var(--section-alt);
+body:last-child {
+  :global(div.ant-tooltip) {
+    display: none;
+  }
+}
+
+body > div > div {
+  :global(div.ant-menu-submenu.ant-menu-submenu-popup.ant-menu) {
+    display: none;
+  }
```

Comment:
would really hope we can use some less generic selectors here

Reply:
Refactored

Reply:
is there a reason we need to use `body` here? makes me a little nervous instead of just using the child's selector itself, since the structure of the page might change in the future